src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,34 @@
+package core.basesyntax;
+
+public class FigureSupplier {
+    private static final int SIDE_BOUND = 20;
+    private static final int ARRAY_BOUND = 5;
+    private static final int DEFAULT_RADIUS = 10;
+    private static final String DEFAULT_COLOR = "WHITE";
+    private final ColorSupplier supplier = new ColorSupplier();
+    private final Figure circle = new Circle(getUnit(), getFigureColor());
+    private final Figure square = new Square(getUnit(), getFigureColor());
+    private final Figure rectangle = new Rectangle(getUnit(), getUnit(), getFigureColor());
+    private final Figure rightTriangle = new RightTriangle(getUnit(), getUnit(), getFigureColor());
+    private final Figure isoscelesTrapezoid = new IsoscelesTrapezoid(getUnit(), getUnit(),
+            getUnit(), getFigureColor());
```

Comment:
don’t make this variable a class field.
The class fields should represent the state of the class objects otherwise they should be local variables.